src/renate/updaters/experimental/er.py

```diff
@@ -95,7 +95,7 @@ def train_dataloader(self) -> DataLoader:
         )
         return DataLoader(
             train_dataset,
-            batch_size=self._batch_size,
```

Comment:
Nit: Would it make sense to add a check for the value of `self._batch_memory_frac`? 
I.e., add as statement of the form 
`assert 0 <= self._batch_memory_frac <= 1.`

Reply:
Yes, good point. Will add this.